### PR TITLE
add confusion matrix for temporal training and sort labels name alphabetically

### PR DESCRIPTION
**Pull request by yoga-0125**
- sort `label_names` in `train_classifier.py`
- generate confusion matrix for temporal training in `finetuning.py`

### CODE REVIEW

File: tools/train_classifier.py [hunk 1]
```diff
@@ -124,6 +124,7 @@ def train_model(path_in, path_out, model_name, model_version, num_layers_to_fine
 
     # Find label names
     label_names = os.listdir(directories.get_videos_dir(path_in, 'train'))
+    label_names.sort()
```
Comment (YasheshSavani) [inline tools/train_classifier.py:127]: if possible  use `natural sorting` instead of basic sorting.  For that  we have a  library called `natsort`. You can find that in `testing.py` on line 49.

File: sense/finetuning.py [hunk 2]
```diff
@@ -399,7 +400,8 @@ def run_epoch(data_loader, net, criterion, optimizer=None, use_gpu=False,
     top1 = np.mean(epoch_labels == epoch_top_predictions)
     loss = running_loss / len(data_loader)
 
-    cnf_matrix = confusion_matrix(epoch_labels, epoch_top_predictions)
+    cnf_matrix = confusion_matrix(epoch_labels, epoch_top_predictions,
+                                  labels=range(min(epoch_labels), max(epoch_labels) + 1))
```
Comment (corneliusboehm) [inline sense/finetuning.py:404]: Would it be safer to go from 0 to `len(label_names_temporal)` just in case the epoch doesn't contain an example for one of those endpoints?

**Review by corneliusboehm — APPROVED** [review]
Working well :+1: Next we should see if we can use a better metric than accuracy for event detection.
